src/domain/ai_model.py

```diff
@@ -7,9 +7,9 @@
 import cv2
 from PIL import Image
 import numpy as np
-from PytorchWildlife.models import detection as pw_detection
 from PytorchWildlife.data import transforms as pw_trans
 #import intel_npu_acceleration_library as npu_lib
+from domain.openvino_models import OVMegaDetectorV5
 from PytorchWildlife import utils as pw_utils
 from domain.classify_detections import Classifier, txt_animalclasses
```

Comment:
import cv2
import numpy as np
from PIL import Image
from PytorchWildlife import utils as pw_utils
from PytorchWildlife.data import transforms as pw_trans

from domain.classify_detections import Classifier, txt_animalclasses
from domain.openvino_models import OVMegaDetectorV5

Reply:
@alessandropalla removing from "_PytorchWildlife.models import detection as pw_detection_" won't affect WADS to work in non-intel-npu scenario?

Reply:
No, as long as we use onnx for inference (and imho we should)